crates/sui-indexer/src/handlers/checkpoint_handler.rs

```diff
@@ -269,6 +270,27 @@ where
         }))
     }
 
+    fn derive_object_versions(
```

Comment:
Any reason to do it here instead of inside the committer directly?

Reply:
I tend to group indexing codes, namely codes to get Stored* types in `index_checkpoint` while `committer` persists `Stored.*` into DB.